src/domain/Loan.ts

```diff
@@ -16,16 +16,24 @@ export function updateLoans(pool: Pool, pileAddress: string): BigInt[] {
     let loans = pool.loans
     let loanId = loans[j]
 
+    let loan = Loan.load(loanId)
+    if (!loan) {
+      log.error('updateLoans: loan {} not found', [loanId])
+      return [new BigInt(0), new BigInt(0)]
+    }
+
+    // Ignore closed loans
+    if (loan.closed === 0) {
```

Comment:
Isn't it closed when the value is not 0, so this should be
```suggestion
    if (loan.closed !== 0) {
```